runtime/bcverify/vrfyhelp.c

```diff
@@ -1126,7 +1126,7 @@ parseObjectOrArrayName(J9BytecodeVerificationData *verifyData, U_8 *signature)
 		signature++;
 	}
 	arity = (UDATA) (signature - string);
-	if (*signature == 'L') {
+	if (J9_IS_OBJECT_OR_VALUETYPE(*signature)) {
```

Comment:
This loses the distinction between Q and L which will be problematic for the verifier